applets/brightness/brightness-applet.c

```diff
@@ -941,17 +940,17 @@ gpm_applet_dialog_about_cb (GtkAction *action, gpointer data)
 	license_trans = g_strconcat (_(license[0]), "\n\n", _(license[1]), "\n\n",
 				     _(license[2]), "\n\n", _(license[3]), "\n", NULL);
 
-	about = (MateAboutDialog*) mate_about_dialog_new ();
-	mate_about_dialog_set_program_name (about, GPM_BRIGHTNESS_APPLET_NAME);
-	mate_about_dialog_set_version (about, VERSION);
-	mate_about_dialog_set_copyright (about, _("Copyright \xc2\xa9 2006 Benjamin Canou"));
-	mate_about_dialog_set_comments (about, GPM_BRIGHTNESS_APPLET_DESC);
-	mate_about_dialog_set_authors (about, authors);
-	mate_about_dialog_set_documenters (about, documenters);
-	mate_about_dialog_set_translator_credits (about, translator_credits);
-	mate_about_dialog_set_logo (about, logo);
-	mate_about_dialog_set_license (about, license_trans);
-	mate_about_dialog_set_website (about, "http://www.mate-desktop.org/");
+	about = (GtkAboutDialog*) gtk_about_dialog_new ();
+	gtk_about_dialog_set_program_name (about, GPM_BRIGHTNESS_APPLET_NAME);
+	gtk_about_dialog_set_version (about, VERSION);
+	gtk_about_dialog_set_copyright (about, _("Copyright \xc2\xa9 2011-2016 MATE Developer"));
```

Comment:
I think we can keep previous copyright too. Also there's a small typo, should be ```MATE Developers``` :smile: